test/unit-test/core_json_utest.c

```diff
@@ -1589,4 +1658,13 @@ void test_JSON_unreached( void )
     buf[ 0 ] = 'x';
     start = 0;
     TEST_ASSERT_EQUAL( false, skipEscape( buf, &start, sizeof( buf ) ) );
+
+    /* set output value to -1 when integer conversion exceeds max */
+    {
+#define TOO_BIG    "100000000000"
```

Comment:
Should you also test this value with `JSON_Search`?

Reply:
I think the tests are adequate.  Branch coverage is 100%.